### PR TITLE
Warning fixes and removing dead code

### PR DESCRIPTION
**Pull request by ZappoMan**
No changes in functionality, just cleaning up the code.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 65a79cfefbdf2d9cfc15f83d7f0bf0c31eed799c available at: https://gist.github.com/7ece9b059ef554f26f87<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5374.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5374.dmg

**Comment by ZappoMan** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 65a79cfefbdf2d9cfc15f83d7f0bf0c31eed799c available at: https://gist.github.com/7ece9b059ef554f26f87<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5374.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5374.dmg

File: interface/src/ui/ApplicationOverlay.cpp [hunk 1]
```diff
@@ -32,8 +32,8 @@
 
 const vec4 CONNECTION_STATUS_BORDER_COLOR{ 1.0f, 0.0f, 0.0f, 0.8f };
 const float CONNECTION_STATUS_BORDER_LINE_WIDTH = 4.0f;
-static const float ORTHO_NEAR_CLIP = -10000;
-static const float ORTHO_FAR_CLIP = 10000;
+static const float ORTHO_NEAR_CLIP = -1000;
+static const float ORTHO_FAR_CLIP = 1000;
```
Comment (Atlante45) [inline interface/src/ui/ApplicationOverlay.cpp]: Should be *.0f

File: interface/src/Stars.cpp [hunk 2]
```diff
@@ -31,14 +31,14 @@
...
-static const float HALF_TAU = TAU / 2.0f;
-static const float QUARTER_TAU = TAU / 4.0f;
-static const float MILKY_WAY_WIDTH = TAU / 30.0f; // width in radians of one half of the Milky Way
-static const float MILKY_WAY_INCLINATION = 0.0f; // angle of Milky Way from horizontal in degrees
-static const float MILKY_WAY_RATIO = 0.4f;
+//static const float HALF_TAU = TAU / 2.0f;
+//static const float QUARTER_TAU = TAU / 4.0f;
+//static const float MILKY_WAY_WIDTH = TAU / 30.0f; // width in radians of one half of the Milky Way
+//static const float MILKY_WAY_INCLINATION = 0.0f; // angle of Milky Way from horizontal in degrees
+//static const float MILKY_WAY_RATIO = 0.4f;
```
Comment (Atlante45) [inline interface/src/Stars.cpp]: Didn't we agreed to remove dead code instead of commenting it?

Comment (ZappoMan) [inline interface/src/Stars.cpp]: I left these in because there's a FIXME comment above to use these parameters and add star colors.

**Pull request merged**